### PR TITLE
[4618][3.1.x] Add ability to include TinyMCE plugins #2499

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4618

### CODE REVIEW

File: src/main/webapp/repo-bootstrap/global/configuration/samples/sample-form-control-rte-setup-tinymce5.xml [hunk 1]
```diff
@@ -44,10 +44,14 @@
...
         </toolbarItems1>
-        <toolbarItems2></toolbarItems2>
+        <toolbarItems2>my_button</toolbarItems2>
         <toolbarItems3></toolbarItems3>
         <toolbarItems4></toolbarItems4>
-        
+
+        <external_plugins>
+            <my_button><![CDATA[/studio/api/2/plugin/file?siteId=mysite&type=tinymce&name=my_button&filename=plugin.min.js]]></my_button>
+        </external_plugins>
```
Comment (rart) [inline src/main/webapp/repo-bootstrap/global/configuration/samples/sample-form-control-rte-setup-tinymce5.xml:53]: Wrong url & not commented

File: src/main/webapp/repo-bootstrap/global/configuration/samples/sample-form-control-rte-setup-tinymce5.xml [hunk 2]
```diff
@@ -44,10 +44,14 @@
         <toolbarItems1>
             formatselect | bold italic strikethrough forecolor backcolor | link | alignleft aligncenter alignright alignjustify | numlist bullist outdent indent | removeformat
         </toolbarItems1>
-        <toolbarItems2></toolbarItems2>
+        <toolbarItems2>my_button</toolbarItems2>
```
Comment (rart) [inline src/main/webapp/repo-bootstrap/global/configuration/samples/sample-form-control-rte-setup-tinymce5.xml:47]: rollback

**Review by rart — CHANGES_REQUESTED** [review]
See https://github.com/craftercms/craftercms/issues/4618#issuecomment-814146412

File: src/main/webapp/repo-bootstrap/global/configuration/samples/sample-form-control-rte-setup-tinymce5.xml (hunk 1 above)
Comment (jvega190) [reply src/main/webapp/repo-bootstrap/global/configuration/samples/sample-form-control-rte-setup-tinymce5.xml:53]: the url didn't change in 3.1.x

**Review by rart — APPROVED** [review]

**Pull request merged**